### DISCUSSION TITLE
Has Docker config has been removed from edgedb.com/docs?

### DISCUSSION THREAD

**Discussion by agritheory** [geldata/gel#1695] [General]
I cannot seem to find the documentation that indicates how mount a volume and docker container. Searching the docs only yields the GraphQL+Docker port caveat. This seems worth of its own section, perhaps in the Administration section?

**Comment by tailhook** [comment]
We're working on `edgedb server install --method=docker` that would allow install and manage dockerized instances locally as easy as non-dockerized installations. Does it solve your issue?

Currently, you can mount any directory and use that for EdgeDB when running server `edgedb-server --data-dir=/mounted/volume`

Other than that, we probably need a section on edgedb server command-line arguments in the Administration section.

**Reply by agritheory** [reply]
It solves my issue but prompts another question: how does `edgedb server` interface with docker-compose, Portainer, K8s or any other abstracted container utility?

**Reply by tailhook** [reply]
This is a good question. For docker compose we're probably going to show YAML snippet. But still support upgrading such instances.

For K8s we definitely need more documentation. I have no experience with Portainer, but we are going to have low-level documentation anyway, so you can integrate EdgeDB into other systems.
